.github/workflows/test-lint.yml

```diff
@@ -1,29 +0,0 @@
-name: Lint Test
```

Comment:
Can we keep this file?

Reasons:
- It is small and easy to read/understand
- It runs in 15sec [so no "latency issues" ]
- It can be used aside from the other big CI file "build.yml" which is harder to understand and which can be used only for release etc.

Reply:
We can convert it to a reusable script and launch it. I can do that. Otherwise the content of the actions was already run except for the lint but it was failing.

Reply:
There is a problem. When converting it to a reusable script it will still run within the other workflow. So essentially the other workflow steps cannot gain access of the results of the extra workflow. So all steps have to be done in both places to prevent the build from happening. In the failing case the whole workflow should only take 15seconds.

Reply:
I will convert it so you can see how it looks.

Reply:
It looks way better than I thought it was going to look. The workflow shows up as a normal dependency
![image](https://user-images.githubusercontent.com/708186/230611680-a002e267-4a5e-491e-9126-22ee738ec92c.png)

Reply:
Can we close this conversation? It's now using the original file you had, but as a direct reusable task.